apps/sms/views/inbox/js/inbox.js

```diff
@@ -986,6 +987,12 @@ var InboxView = {
     } else {
       this.updateThread(thread);
     }
+
+    // If the draft we scheduled notification for has been deleted, we shouldn't
+    // notify user anymore.
+    if (draft.id === this.notifyAboutSavedDraftWithId) {
```

Comment:
__question:__ just for the sake of complete picture, I think about very-very edge case here:
* User enters existing conversation;
* Enters some text to input field or adds attachment - saves draft;
* Then long-presses on any message and chooses forward;
* Enters some text there as well - taps on Home button to save this draft as well;
* And then removes everything (discards draft) and presses back to return to Inbox.

After this PR user will not see "draft saved" notification for the draft saved in conversation, but I believe otherwise it would be confusing, user has probably already lost context related to the conversation :) What do you think?